lib/pages/chat/chat.dart

```diff
@@ -1284,6 +1284,9 @@ class ChatController extends State<Chat>
   void onSendFileClick(BuildContext context) async {
     if (PlatformInfos.isMobile) {
       _showMediaPicker(context);
+    } else if (PlatformInfos.isDesktop) {
```

Comment:
should it be isLinux only or `isDesktop`, i see you use isLinux somewhere

Reply:
maybe you should make is method in `PlatformInfo`

Reply:
since this condition calls specific desktop methods, not limited to linux (check  `pickFilesFromDesktop`) it is more relevant to use `isDesktop` here IMO